components/Hero/index.vue

```diff
@@ -1,27 +1,52 @@
-<script setup lang="ts"></script>
-
 <template>
   <div class="main">
-    <HeroIntroText v-if="!hideSection" />
-    <div class="flex-grow">
-      <HeroWaitlist @hide-intro="(value) => (hideSection = value)" />
-      <div class="lg:hidden">
-        <a href="#learn-more" class="link" v-if="!hideSection">
-          {{ $t('link.learnMore') }}
+    <div class="w-full">
+      <div class="w-3/4">
+        <h1 class="text text-6xl lg:text-8xl">
+          {{ $t('homeHero.title') }}
+        </h1>
+        <h2 class="text my-5 text-3xl lg:text-4xl">
+          {{ $t('homeHero.intro') }}
+        </h2>
+        <a :href="localePath('/dashboard')">
+          <button class="btn">
+            {{ $t('homeHero.tryItOut') }}
+            <IconArrowRight class="mx-2 rtl:hidden ltr:inline-block" />
```

Comment:
Load the SVGs as images instead of calling them as components.
Also use CSS to flip a single image, not two images.